src/scripts/main.js

```diff
@@ -1,3 +1,45 @@
 'use strict';
 
-// write code here
+const list = document.body.querySelectorAll('li');
+
+function sortList(items) {
+  const sorted = Array.from(list).sort((a, b) => {
```

Comment:
The `sortList` function is using the `list` variable directly instead of the `items` parameter passed to the function. Consider replacing `list` with `items` to use the function parameter.